book/src/generated/typable-cmd.md

```diff
@@ -63,5 +63,6 @@
 | `:tree-sitter-subtree`, `:ts-subtree` | Display tree sitter subtree under cursor, primarily for debugging queries. |
 | `:config-reload` | Refreshes helix's config. |
 | `:config-open` | Open the helix config.toml file. |
+| `:log-open` | Open the helix log file. |
```

Comment:
I am not aware if this is from other editors? Why do we want to do this rather something like variable `$VIMRUNTIME`?

Reply:
> I am not aware if this is from other editors? Why do we want to do this rather something like variable `$VIMRUNTIME`?

IMO, as the "post-modern" editor, I think Helix's point is exactly to have some features that other editors doesn't have.
Although read it from a variable is pretty handy too!